### PR TITLE
Testing Info added in Readme

### PR DESCRIPTION
**Pull request by rjt-gupta**
Closes - #195

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Getting started
 
 You obviously want to bind to 0.0.0.0 and port 80 when running in <i>production</i>.
 
+## Testing
+
+For testing speedily along with full coverage report, we recommend running `pytest` on tests folder:
+
+
+    cd snare/snare
```
Comment (glaslos) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ Getting started
 
 You obviously want to bind to 0.0.0.0 and port 80 when running in <i>production</i>.
 
+## Testing
+
+For testing speedily along with full coverage report, we recommend running `pytest` on tests folder:
```
Comment (glaslos) [inline README.md]:
```suggestion
In order to run the tests and receive a test coverage report, we recommend running `pytest`:
```

Comment (rjt-gupta) [reply README.md]: Thanks for the review, will do the changes shortly :)

**Comment by rjt-gupta** [comment]
This is ready to review.

File: README.md [hunk 3]
```diff
@@ -39,6 +39,14 @@ Getting started
 
 You obviously want to bind to 0.0.0.0 and port 80 when running in <i>production</i>.
 
+## Testing
+
+In order to run the tests and receive a test coverage report, we recommend running `pytest`:
+
+
+    sudo pytest --cov-report term-missing --cov=snare tests/
```
Comment (glaslos) [inline README.md]:
```suggestion
    pip install pytest pytest-cov
    sudo pytest --cov-report term-missing --cov=snare snare/tests/
```

Comment (glaslos) [reply README.md]: Let's add some information on how to install dependencies and the complete test path assuming run from the root folder

Comment (rjt-gupta) [reply README.md]: Much Better! Thanks

**Comment by rjt-gupta** [comment]
Anything else that needs to improve?

**Pull request merged**